include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 13
+#define SOF_ABI_MINOR 14
```

Comment:
that seems wrong based on @singalsu changes?

Reply:
@akloniex According to ABI classifier decision, this should be ABI 16:
https://github.com/orgs/thesofproject/projects/2
... but this patch and firmware https://github.com/thesofproject/sof/pull/2580 are now at ABI 14. Which is correct, ABI classifier or the patches? @lgirdwood

Reply:
@kv2019i ABI updated as assigned in classifier.